src/app/services/api-data.service.ts

```diff
@@ -244,6 +311,25 @@ export class ApiDataService {
     );
   }
 
+  getAllArticlesForProject(project: string) {
+    const url = this.hostSetting.host + 'article/_search/?size=100000&q=secondaryProject:' + project;
+    const headers = new HttpHeaders().set('Content-Type', 'application/json');
```

Comment:
same thing here, headers is not required.

Reply:
done in c257a2a

Reply:
getAllArticlesForProject still has headers option in request, can you remove it please